.buildkite/package-lock.json

```diff
@@ -369,7 +369,7 @@
     "node_modules/kibana-buildkite-library": {
```

Comment:
Can you remove this file from the PR? I think it's left over from when you had package.json changes

Reply:
Are you sure? This file isn't added in this PR and we definitely use it

Reply:
Sorry, I meant remove the changes, not the whole file

Reply:
Yeah, I recommend running

```
git checkout upstream/main -- .buildkite/package-lock.json
```

Reply:
Roger that, thanks gents.